apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -277,4 +464,339 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
   test "empty processor returns no exiting utxo positions", %{processor_empty: empty} do
     assert [] = Core.get_exiting_utxo_positions(empty)
   end
+
+  @tag fixtures: [:processor_empty]
+  test "empty processor returns no in flight exits", %{processor_empty: empty} do
+    assert %{} == Core.get_in_flight_exits(empty)
+  end
+
+  @tag fixtures: [:processor_empty, :in_flight_exit_events, :contract_ife_statuses, :in_flight_exits]
+  test "properly processes new in flight exits", %{
+    processor_empty: empty,
+    in_flight_exit_events: events,
+    contract_ife_statuses: statuses,
+    in_flight_exits: ifes
+  } do
+    {updated_state, _} = Core.new_in_flight_exits(empty, events, statuses)
+
+    assert ifes
+           |> Map.new() == Core.get_in_flight_exits(updated_state)
```

Comment:
Please stop pipe operator abuse. Sign the petition!

Reply:
I will ! :floppy_disk: :whale: